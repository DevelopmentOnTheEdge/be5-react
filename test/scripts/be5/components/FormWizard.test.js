import React          from 'react';
import renderer       from 'react-test-renderer';
import FormWizard     from '../../../../src/scripts/be5/components/FormWizard';
import be5            from '../../../../src/scripts/be5/be5';


test('test', () => {
  const steps = [
    {title: "Companies", url: '#!text/A'},
    {title: "EgissoDocs", url: '#!text/B'},
  ];

  const component = renderer.create(
    <FormWizard steps={steps}
                backButtonText={be5.messages.backButtonText}
                nextButtonText={be5.messages.nextButtonText}
    />
  );

  expect(component.toJSON()).toMatchSnapshot();

  //changeDocument('MainDocument', { value: "Page loaded." } );
  //expect(component.toJSON()).toMatchSnapshot();
});
