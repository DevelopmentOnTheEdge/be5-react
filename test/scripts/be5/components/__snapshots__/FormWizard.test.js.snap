// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test 1`] = `
<div
  className="formWizard"
  onKeyDown={[Function]}
>
  <ol
    className="progtrckr clearfix"
  >
    <li
      className="progtrckr-doing"
      onClick={[Function]}
      value={0}
    >
      <em>
        1
      </em>
      <span
        dangerouslySetInnerHTML={
          Object {
            "__html": "Companies",
          }
        }
      />
    </li>
    <li
      className="progtrckr-todo"
      onClick={[Function]}
      value={1}
    >
      <em>
        2
      </em>
      <span
        dangerouslySetInnerHTML={
          Object {
            "__html": "EgissoDocs",
          }
        }
      />
    </li>
  </ol>
  <div
    className="document-content"
    id="document-content___FormWizard"
  >
    <h1>
      Unknown action: companies
    </h1>
  </div>
  <br />
  <div
    className="footer-buttons"
    style={Object {}}
  >
    <button
      className="btn btn-next btn-primary pull-left disabled"
      id="prev-button"
      onClick={[Function]}
    >
      Previous
    </button>
     
    <button
      className="btn btn-prev btn-primary pull-right"
      id="next-button"
      onClick={[Function]}
    >
      Next
    </button>
  </div>
</div>
`;
